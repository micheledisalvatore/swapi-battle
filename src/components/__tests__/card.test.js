import React from 'react';
import { render } from '@testing-library/react';
import dataContext from '../../contexts/data-context'
import Card from '../card'

test('renders the winner component', () => {
  const { container } = render(<dataContext.Provider value={{ attribute: 'foo' }}><Card player={{ name: 'bar name', foo: 100 }} winner={true} /></dataContext.Provider>)

  expect(container).toMatchSnapshot()
})
