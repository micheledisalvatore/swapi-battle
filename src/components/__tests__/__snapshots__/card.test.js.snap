// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the winner component 1`] = `
<div>
  <div
    class="card text-white bg-success false"
  >
    <h5
      class="card-header"
    >
      bar name
    </h5>
    <div
      class="card-body"
    >
      <button
        class="btn btn-primary"
        type="button"
      >
        foo
         
        <span
          class="badge badge-light"
        >
          100
        </span>
      </button>
    </div>
  </div>
</div>
`;
